arelle/plugin/validate/NL/rules/fg_nl.py

```diff
@@ -35,6 +35,11 @@
     'fact': 6,
 }
 
+VALID_UTR_NAMESPACES = [
+    'http://www.xbrl.org/2003/instance',
+    'http://www.xbrl.org/2009/utr',
+    'http://www.xbrl.org/2003/iso4217'
+]
```

Comment:
Going to be a flake8 issue here, need to retain two lines of whitespace above function definition.